src/wazuh_modules/vulnerability_scanner/tests/component/vulnerabilityScannerFacade_test.cpp

```diff
@@ -44,7 +46,7 @@ TEST_F(VulnerabilityScannerFacadeTest, TestSingletonOfTheVulnerabilityScannerFac
 /*
  * @brief Test start method of the VulnerabilityScannerFacade class.
  */
-TEST_F(VulnerabilityScannerFacadeTest, TestStartMethod)
+TEST_F(VulnerabilityScannerFacadeTest, TestStartToEndMethod)
 {
     // TODO: Remove GTEST_SKIP and add EXPECTS once the implementation of the 'Indexer Connector' module is completed
     GTEST_SKIP();
```

Comment:
This test is disabled due to this, should we remove this line?

Reply:
We should keep it